app/controllers/admin/radios_controller.rb

```diff
@@ -91,6 +91,7 @@ def radio_params
         :digest_mp3,
         community_ids: [],
         personality_ids: [],
+        comic_attributes: :image
```

Comment:
__[RuboCop]__

Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.


<!-- Comment by [Sider](https://sider.review) -->